recipe/meta.yaml

```diff
@@ -37,7 +37,7 @@ requirements:
   host:
     - python
     - pip
-    - maturin >=0.11.4,<0.12
+    - maturin >=0.12.4,<0.13
```

Comment:
Small issue but in the upstream `pyproject.toml` it says `>=0.12.3,<0.13`

Reply:
Indeed, I'll file a PR there as well, but since this is a build time dependency it might not block this feedstock PR. What do you think?

Reply:
I think @conda-forge/maturin should be asked to provide a `py310` build of maturin for `v0.11.x` series.

Reply:
What's the point for keeping the previous version for a build time dependency?

Reply:
Ah, looks like upstream has already made the jump https://github.com/ijl/orjson/commit/74da2ff8602e33352df3508113684363d25641d0